### PR TITLE
Add more sounds before "go live"

### PR DESCRIPTION
**Pull request by gkhays**
The installation would benefit from additional sound assets. This pull request adds several new sounds from a royalty free library. See [Pixabay](https://pixabay.com/sound-effects/search/industrial/). **Note**: these sounds have been trimmed to be 15 seconds or less in duration.

Side note: the [angry horn](https://pixabay.com/sound-effects/traimory-mega-horn-angry-siren-f-cinematic-trailer-sound-effects-193408/) effect didn't make the cut but I really like it!

New assets include a riveted metal sheet that could be used as a background for the UI.

There is also some JavaScript to manipulate button visibility that didn't get used but may prove useful going forward.

## References
- [Pixabay royalty-free sounds](https://pixabay.com/sound-effects/search/industrial/)

### CODE REVIEW

File: pod/ops.py
Comment (Zenguin15823) [inline pod/ops.py:1]: Goodbye, "hello.mp3," how sad to see you go...

File: pod/static/images/bg-rivet.jpg
Comment (Zenguin15823) [inline pod/static/images/bg-rivet.jpg:1]: Woah!

File: pod/templates/index.html
Comment (Zenguin15823) [inline pod/templates/index.html:1]: Stylish!

**Review by Zenguin15823 — APPROVED** [review]

File: pod/ops.py
Comment (gkhays) [reply pod/ops.py:1]: The boss said no human voices. We have to add some AI for that!

File: pod/static/images/bg-rivet.jpg
Comment (gkhays) [reply pod/static/images/bg-rivet.jpg:1]: Keanu, is that you?

**Pull request merged**